Makefile

```diff
@@ -43,11 +46,9 @@ $(call verify-golang-versions,Dockerfile)
 
 $(call add-crd-gen,kueueoperator,./pkg/apis/kueueoperator/v1alpha1,./manifests/,./manifests/)
 
-test-e2e: GO_TEST_PACKAGES :=./test/e2e
-# the e2e imports pkg/cmd which has a data race in the transport library with the library-go init code
-test-e2e: GO_TEST_FLAGS :=-v
-test-e2e: test-unit
 .PHONY: test-e2e
+test-e2e: deploy-cert-manager
```

Comment:
I suggest removing the dependency on deploy-cert-manager here. So you can run `make deploy-cert-manager test-e2e`.

Reply:
done.